mealie/services/email/email_senders.py

```diff
@@ -24,8 +24,10 @@ def send(self, email_to: str, subject: str, html: str) -> bool:
         )
 
         smtp_options: dict[str, str | bool] = {"host": self.settings.SMTP_HOST, "port": self.settings.SMTP_PORT}
-        if self.settings.SMTP_TLS:
```

Comment:
Is there a good way to test the DefaultEmailSender? I see the test makes its own implementation and I don't want to rewrite the production code in the test...

Reply:
I usually just use the UI on the settings page when I'm testing the mail server that it actually sends mail.